python-avocado.spec

```diff
@@ -1,8 +1,24 @@
 %global srcname avocado
+
+# Conditional for release vs. snapshot builds. Set to 1 for release build.
+%if ! 0%{?rel_build:1}
```

Comment:
Would it be possible to change this to `release_build`? The `rel_build` is quite a cryptic name... (not a strong requirement, though)